db/seeds.rb

```diff
@@ -1,7 +1,171 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+puts "Cleaning Database 🧼"
+sleep 1
+print " ."
+sleep 1
+print " ."
+sleep 1
+puts " ."
+sleep 1
+puts ""
+
+# Clear database to prevent duplicate records
```

Comment:
Some of these can be removed, if they are on the User right? cause of the dependent destroy

Reply:
I had to leave the Friendships because of the double user in there 👯